bpf/bpf_lxc.c

```diff
@@ -1243,9 +1257,14 @@ ipv4_policy(struct __ctx_buff *ctx, int ifindex, __u32 src_label, __u8 *reason,
 	send_trace_notify4(ctx, TRACE_TO_LXC, src_label, SECLABEL, orig_sip,
 			   LXC_ID, ifindex, *reason, monitor);
 
+#if !defined(ENABLE_ROUTING) && defined(ENCAP_IFINDEX) && !defined(ENABLE_NODEPORT)
+	/* See comment in IPv6 path. */
```

Comment:
Nit: I think a consensus is to document IPv4 path, and refer IPv6 to IPv4 for comments.